subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -280,9 +279,7 @@ static int parse_recv_state(const void *data, uint16_t length,
 			return BT_GATT_ERR(BT_ATT_ERR_INVALID_ATTRIBUTE_LEN);
 		}
 
-		broadcast_code = net_buf_simple_pull_mem(&buf, BT_ISO_BROADCAST_CODE_SIZE);
-		(void)memcpy(recv_state->bad_code, broadcast_code,
-			     sizeof(recv_state->bad_code));
+		memset(recv_state->bad_code, 0xFF, sizeof(recv_state->bad_code));
```

Comment:
This change should be reverted. The change is for the scan delegator /broadcast sink only